File: paradigmas de programação/java/lojinha.java
//diretório da pasta onde o arquivo está
package lojinha;
//inicio do programa

//classe principal da lojinha
class Lojinha {
//Método principal, o ponto de entrada do programa
    public static void main(String[] args) {
//Mensagem de boas vindas, para ficar bonitinho
        System.out.println("\n************************************");
        System.out.println("***Bem-Vindo(a) à nossa lojinha!!***");
        System.out.println("************************************");
//mensagens sobre os produtos no estoque atual
        System.out.println("\n////////////////////////////////////////////////////////////////////");
        System.out.println("///Esses são os produtos que constam no nosso estoque atualmente///");
        System.out.println("///////////////////////////////////////////////////////////////////");
//importando o método do arquivo lojinha_produtos.java para exibir os produtos do estoque atual
        lojinha_produtos.ver_produtos();
//mensagem sobre os produtos que estão para chegar no estoque
        System.out.println("\n//////////////////////////////////////////////////////////////////");
        System.out.println("///Esses são os produtos novos que estão para chegar no estoque///");
        System.out.println("//////////////////////////////////////////////////////////////////");
//importando o método do arquivo lojinha_novos.java para exibir os produtos que estão para chegar no estoque
        Lojinha_novos.ver_novos();
    }
}
//fim do programa

File: paradigmas de programação/python/lojinha.py
#inicio do programa

# Classe para exibir produtos do estoque
class exibir_produtos_do_estoque:
    def exibir_produtos_do_estoque():
        # Importando o módulo produtos_lojinha
        import produtos_lojinha

# Classe para exibir produtos novos do estoque
class exibir_produtos_novos_do_estoque:
    def exibir_produtos_novos_do_estoque():
        # Importando o módulo produtos_novos
        import produtos_novos

# Classe principal Lojinha
class lojinha:
# Mensagens de boas-vindas e informações sobre os produtos no estoque
    print("\n************************************")
    print("***Bem-Vindo(a) à nossa lojinha!!***")
    print("************************************")
    print("\n////////////////////////////////////////////////////////////////////")
    print("///Esses são os produtos que constam no nosso estoque atualmente///")
    print("///////////////////////////////////////////////////////////////////")

# Chamando o método para exibir os produtos do estoque
    exibir_produtos_do_estoque.exibir_produtos_do_estoque()

# Mensagens sobre os produtos novos que estão para chegar
    print("\n//////////////////////////////////////////////////////////////////")
    print("///Esses são os produtos novos que estão para chegar no estoque///")
    print("//////////////////////////////////////////////////////////////////")

# Chamando o método para exibir os produtos novos do estoque
    exibir_produtos_novos_do_estoque.exibir_produtos_novos_do_estoque()
#fim do programa